test/e2e/tests/dapp-interactions.spec.js

```diff
@@ -0,0 +1,125 @@
+const { strict: assert } = require('assert');
+const {
+  convertToHexValue,
+  withFixtures,
+  connectDappWithExtensionPopup,
+} = require('../helpers');
+
+describe('Dapp interactions', function () {
+  let windowHandles;
+  let extension;
+  let popup;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('should a notification be triggered despite Metamask being locked', async function () {
```

Comment:
Nit: can we rename the test so it flows when you read the `it` along with the test name e.g. ... It `should trigger the add chain confirmation despite MetaMask being locked`

Reply:
@PeterYinusa fixed